PWGHF/HFC/Tasks/femtoDreamPairTaskCharmHadrons.cxx

```diff
@@ -357,6 +405,47 @@ struct femtoDreamPairTaskCharmHadrons {
     }
   }
   PROCESS_SWITCH(femtoDreamPairTaskCharmHadrons, processMixedEvent, "Enable processing mixed events", true);
+
+  /// process function for to call doSameEvent with Monte Carlo
+  /// \param col subscribe to the collision table (Monte Carlo Reconstructed reconstructed)
+  /// \param parts subscribe to joined table FemtoDreamParticles and FemtoDreamMCLables to access Monte Carlo truth
+  /// \param FemtoDreamMCParticles subscribe to the Monte Carlo truth table
+  void processSameEventMc(FilteredMcColision& col,
+                          o2::aod::FDMCCollisions&,
```

Comment:
is this collision table really needed? Still, it's of different type w.r.t. `col`

Reply:
removed